CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.16.2 (2021-04-30)
+
+- Fixed input placeholder text styles in modern theme. [#226](https://github.com/blackbaud/skyux-theme/pull/226)
```

Comment:
```suggestion
- Fixed placeholder text styles in modern theme. [#226](https://github.com/blackbaud/skyux-theme/pull/226)
```
We just refer to it as "placeholder text" in our guidelines, so I don't think we need "input" here. No big deal if you want to keep it though.